src/platforms/java/migration.mdx

```diff
@@ -4,6 +4,10 @@ sidebar_order: 1000
 description: "Learn about migrating from io.sentry:sentry 1.x to io.sentry:sentry 3.x."
 ---
 
+## Migrating from `io.sentry:sentry` `4.3.0` to `io.sentry:sentry` `5.0.0`
+
+`Sentry#startTransaction` by default does not bind created transaction to the scope. To start transaction with binding to the scope, use one of the new overloaded `startTransaction` methods taking `bindToScope` parameter and set it to `true`. Bound transaction can be retrieved with `Sentry#getSpan`.
```

Comment:
should also be added to platforms/android/migration/

Reply:
Why in a separate PR?

Reply:
same reason as https://github.com/getsentry/sentry-docs/pull/3293#discussion_r626398585